templates/_helpers.tpl

```diff
@@ -203,7 +203,7 @@ Source: https://stackoverflow.com/a/52024583/3027614
 {{- $store := index . 1 -}}
 {{- $storeConfig := index $global.Values.server.config.persistence $store -}}
 {{/* Cassandra password is optional, but we will create an empty secret for it */}}
-{{- print "password" -}}
+{{- default "password" $storeConfig.cassandra.existingSecretKey -}}
```

Comment:
This syntax is backwards, the default value comes second.

Reply:
No, it doesn't.

http://masterminds.github.io/sprig/defaults.html